src/utils.ts

```diff
@@ -1,12 +1,12 @@
-export function changePathVisibility(path: string, hide: boolean) {
+export async function changePathVisibility(path: string, hide: boolean) {
```

Comment:
This function isn't doing anything asynchronous, so it should remain synchronous unless something added to it makes it do asynchronous operations.